zcash_client_sqlite/src/prepared.rs

```diff
@@ -516,19 +561,17 @@ impl<'a, P: consensus::Parameters> DataConnStmtCache<'a, P> {
     pub(crate) fn stmt_insert_address(
         &mut self,
         account: AccountId,
-        mut diversifier_index: DiversifierIndex,
+        diversifier_index: DiversifierIndex,
         address: &UnifiedAddress,
-    ) -> Result<i64, SqliteClientError> {
-        diversifier_index.0.reverse();
-        let sql_args: &[(&str, &dyn ToSql)] = &[
-            (":account", &u32::from(account)),
-            (":diversifier_index_be", &&diversifier_index.0[..]),
-            (":address", &address.encode(&self.wallet_db.params)),
-        ];
-
-        self.stmt_insert_address.execute(sql_args)?;
+    ) -> Result<(), SqliteClientError> {
+        self.stmt_insert_address.execute(
+            &self.wallet_db.params,
+            account,
+            diversifier_index,
+            address,
+        )?;
 
-        Ok(self.wallet_db.conn.last_insert_rowid())
+        Ok(())
```

Comment:
I confirmed that the return value of this function wasn't being used for anything.